### DISCUSSION TITLE
Can set a block name

### DISCUSSION THREAD

**Discussion by neerfix** [postmanlabs/postman-flows#121] [Ideas]
When you're many block, it's hard to know, who does what.

With block name, we are this result : 
![Screenshot_2021-11-12_at_9.56.35_AM](https://i.ibb.co/p18KDRV/Screenshot-2021-11-12-at-9-56-35-AM.png)

**Comment by saswatds** [comment]
Hey @Neerfix , thanks for the feedback. We will add this soon.
